tsconfig.json

```diff
@@ -13,5 +13,6 @@
     "sourceMap": true,
     // Docs - https://www.typescriptlang.org/tsconfig#noImplicitAny
     "noImplicitAny": false
-  }
+  },
+  "exclude": ["./src/**/*.spec.ts", "./src/**/*.spec.tsx", "node_modules"]
```

Comment:
Code looks working even without this `exclude`

Reply:
Done